admin/includes/languages/lang.english.php

```diff
@@ -555,7 +555,7 @@
     'TEXT_LETTERS_FREE' => ' Letter(s) free ',
     'TEXT_LINKED_PRODUCTS' => 'Linked Products:',
     'TEXT_MASTER_CATEGORIES_ID' => 'Product Master Category:',
-    'TEXT_MFA_EMAIL_BODY' => "Here is your Admin login verification code:\n\n%s\n\nThis code is for the login which was attempted via IP address: %s\n\nIf this was not you, please change your Admin password promptly.",
+    'TEXT_MFA_EMAIL_BODY' => "Here is your Admin login verification code:\n\n%1$s\n\nThis code is for the login which was attempted via IP address: %2$s\n\nIf this was not you, please change your Admin password promptly.",
```

Comment:
Both `%1$s` and `$2$s` are within double-quoted strings, so PHP will be looking for a `$s` variable to substitute in each.

Reply:
Thanks!